.github/workflows/CompatHelper.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Pkg.add("CompatHelper")
-        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
+        run: julia -e 'using Pkg; Pkg.add("CompatHelper"); Pkg.Registry.status()'
```

Comment:
This does not survive the next commit, so probably this commit can be completely droppen?